### PR TITLE
CT-802 Add algo and add newkey command

### PR DESCRIPTION
**Pull request by argjv**
In this PR:
- Add a command to generate new key pairs like in `bitgo-cli`.
- Add Algorand

### CODE REVIEW

File: package.json
Comment (tylerlevine) [inline package.json]: New version should be 0.10.0

File: package.json
Comment (tylerlevine) [inline package.json]: `^7.0.0` please

File: src/bgcl.js
Comment (tylerlevine) [inline src/bgcl.js]: Don't think you need to pass in external entropy - if you just don't pass any seed, it will create a random one.

**Review by tylerlevine — CHANGES_REQUESTED** [review]
Looks pretty good

File: src/bgcl.js
Comment (argjv) [reply src/bgcl.js]: Should I pass entropy or the seed? I was thinking about making the seed an optional argument so ppl can recover their pub and priv based on that

**Review by tylerlevine — APPROVED** [review]
lgtm, up to you if you'd like to wait for a review from @zacleids or someone with more experience with the cli

**Pull request merged**
